server/controllers/recommendations/replaceCurrentRecommendationController.ts

```diff
@@ -15,14 +15,14 @@ async function get(req: Request, res: Response) {
     })
   ).filter(status => status.active)
 
-  const isClosed = statuses.find(status => status.name === STATUSES.CLOSED)
+  const isClosed = statuses.find(status => status.name === STATUSES.SENT_TO_PPCS)
   const isPPDocumentCreated = statuses.find(status => status.name === STATUSES.PP_DOCUMENT_CREATED)
 
   if (!isClosed && isPPDocumentCreated) {
```

Comment:
I think I've got a similar question here, but in reverse.  Looks like this will set SENT_TO_PPCS for DNTR and Recall.
Or does PP_DOCUMENT_CREATED mean "Part A created" and not "Part A or DNTR created"?